src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+@import 'stars-block';
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  background: $white-color;
+  border-radius: 5px;
+
+  transition: $transition-time all;
+
+  &__img-cover {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+
+    .card__img {
+      width: 100%;
+      height: 100%;
+    }
+  }
+
+  &__title {
+    @include text-styling(12px, 500, 18px);
+
+    transition: $transition-time all;
+
+    margin-top: 40px;
+    margin-bottom: 0;
+  }
+
+  &__descr {
+    @include text-styling(10px, 400, 14px);
+
+    margin-top: 4px;
+
+    color: $light-color;
+  }
+
+  &__reviews {
+    @include text-styling(10px, 400, 14px);
+
+    display: flex;
+    justify-content: space-between;
+    margin-top: 16px;
+
+    .review-descr {
+      text-align: center;
+      display: flex;
+      justify-content: center;
+      align-items: center;
+    }
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+
+    .price-text {
+      @include text-styling(12px, 400, 18px);
+
+      color: $light-color;
+    }
+
+    .price-amount {
+      @include text-styling(16px, 700, 18px);
+
+      font-weight: bold;
+    }
+  }
+
+  &__button {
+    @include text-styling(14px, 700, 16px);
+
+    height: 40px;
+    width: 166px;
+    padding-block: 12px;
+    background: $blue-color;
+    border-radius: 5px;
+    border: 1px solid $blue-color;
+    margin-top: 16px;
+    color: $white-color;
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    transition: $transition-time all;
+    cursor: pointer;
+
+    &:hover {
+      background-color: $white-color;
+      color: $blue-color;
+    }
+  }
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
Using `transform: scale(1.2)` on hover can cause layout shifts, especially in grid or flex layouts. Consider using a smaller scale value or a different effect to avoid disrupting the layout.